<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>408</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Motor controls</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="motorLeft">
         <property name="text">
          <string>Start/Left</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="motorRight">
         <property name="text">
          <string>Start/Right</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="motorSpeedSlow">
         <property name="text">
          <string>Speed/Slow</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="motorSpeedFast">
         <property name="text">
          <string>Speed/Fast</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="stopMotor">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop motor</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gate controls</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="openGate">
         <property name="text">
          <string>Open gate</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="closeGate">
         <property name="text">
          <string>Close gate</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Lamp controls</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="greenOn">
         <property name="text">
          <string>Green ON</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="greenOff">
         <property name="text">
          <string>Green OFF</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="yellowOn">
         <property name="text">
          <string>Yellow ON</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="yellowOff">
         <property name="text">
          <string>Yellow OFF</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="redOn">
         <property name="text">
          <string>Red ON</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="redOff">
         <property name="text">
          <string>Red OFF</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="greenBlinking">
         <property name="text">
          <string>Green blinking ON</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="greenBlinkingOff">
         <property name="text">
          <string>Green blinking OFF</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QPushButton" name="yellowBlinking">
         <property name="text">
          <string>Yellow blinking ON</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QPushButton" name="yellowBlinkingOff">
         <property name="text">
          <string>Yellow blinking OFF</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QPushButton" name="redBlinking">
         <property name="text">
          <string>Red blinking ON</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QPushButton" name="redBlinkingOff">
         <property name="text">
          <string>Red blinking OFF</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Blinking controls</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Connect to:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextBrowser" name="monitorLog"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="controlIP">
         <property name="text">
          <string>192.168.101.109</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="monitoringPort">
         <property name="text">
          <string>8081</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Monitoring log:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Monitoring Port:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="controllerConnectedStatus">
         <property name="text">
          <string>Not connected</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="monitoringStatus">
         <property name="text">
          <string>Listening for connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
